metrics-operator/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/go-logr/logr v1.2.4
 	github.com/gorilla/mux v1.8.0
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/keptn/lifecycle-toolkit/klt-cert-manager v0.0.0-20231004101120-a15b038e1219
+	github.com/keptn/lifecycle-toolkit/klt-cert-manager v0.0.0-20231004144748-2363c9ca439a
```

Comment:
Instead of updating the dependency here, I would rather merge the renovate PR

Reply:
the test will fail otherwise I will anyhow merge the pr after

Reply:
let's merge it, rebase this PR and then the tests should be working as expected